### ISSUE TITLE
VM quick search / selector / sort

### ISSUE THREAD

**Issue by mareklibra** [oVirt/ovirt-web-ui#18]
designs: https://docs.google.com/document/d/1knF5-UnYtM6a8pcPUmnzva2Q5JsBwyOxbgbIXWJ2NAo/edit

![selection_263](https://user-images.githubusercontent.com/2248293/50360507-3b1e8080-052e-11e9-8f00-46bb69860796.png)

**Comment by jelkosz** [comment]
I have tried to use ovirt-web-ui on a larger setup with hundreds of VMs and getting to the VM I wanted to use was a pain (and after about 5 minutes of scroll and load next 8 vms when I was still far from mine I gave up and decided I don't really need to use that VM :) ). 
It would be hugely helpful to have a way how to make a server side filter to return only VMs where the name matches some pattern (as it can be done in webadmin).

Unfortunately, this is supported only for admin users (for a little context check this: https://bugzilla.redhat.com/show_bug.cgi?id=1534607), but it is still useful since mostly the admin users will have hundreds of VMs available so they will appreciate the most this feature.

What I propose is to have a searchbar on top of the screen for admin users. If I write something like tj-* into it, all the VM / pool related queries will only load VMs which match this pattern.

This searchbar will not be shown for regular users.

@mareklibra @jniederm what you guys think?
